manifesto.md

```diff
@@ -0,0 +1,221 @@
+
+Em uma empresa 100% remota, essa dúvida pode se transformar em documentação e então, todo funcionário que venha a ter esse mesmo questionamento conseguirá resolvê-lo de maneira independente.
+
+Esse hábito de converter dúvidas em documentação favorece a criação de um ambiente de trabalho mais eficaz e auto-suficiente.
+
+**Maior visibilidade nos projetos:** A documentação contínua e atualizada proporciona uma visão clara do andamento dos projetos. Isso facilita o acompanhamento, a avaliação e a comunicação do progresso, permitindo que as equipes estejam sempre bem informadas e possam tomar decisões baseadas em dados concretos.
+
+**Agilidade no Onboarding:** Com uma cultura de documentação bem implantada, a agilidade e facilidade de integrar um novo membro da equipe é um efeito colateral natural e muito bem vindo. Ao basear-se em conteúdo textual, seus outros funcionários ficam mais livres para realizar outras tarefas e sua equipe, mais independente.
+
+### Funcionários mais motivados
+
+> Um ambiente de trabalho flexível e bem estruturado aumenta a satisfação e retenção de talentos.
+
+Ao reduzir os gastos com transporte e alimentação e incentivar maior autonomia, responsabilidade e flexibilidade de horário na vida de seus funcionários, uma coisa é certa: **eles serão mais felizes!**
+
+Com maior tempo livre durante a semana (antes gasto unicamente com transporte), é mais fácil manter um equilíbrio entre vida pessoal e trabalho, tornando-os mais dispostos para realizar suas tarefas e valorizar seus empregos.
+
+---
+
+# Nossos Compromissos (Trampar de Casa)
+
+**Promover vagas de qualidade:** Divulgamos vagas 100% remotas, nacionais ou internacionais, de empresas que respeitam e valorizam seus funcionários.
+
+**Aumentar a conscientização sobre o Trabalho Remoto:** analisar e discutir estratégias modernas pensadas para o trabalho remoto, como:
+
+- "Cultura do Recado": _Async First_
+  - Documentação textual
+  - Gravações de vídeo
+  - Preferência por "recados" globais
+    - Facilidade para revisitar decisões, tarefas, projetos e alterações sem envolver um terceiro.
+    - Filosofia Micro: Fragmentação de tarefas em pequenos entregáveis.
+    - Melhoria e aumento da documentação escrita.
+- Liderança baseada em dados e análises estatísticas:
+  - **Data Driven Culture:** Utilizar dados concretos para guiar decisões, otimizando processos e garantindo resultados mais eficientes.
+  > Acompanhar evoluções é melhor do que estimar prazos.
+
+**Promover um ambiente inclusivo**: trabalhar para garantir que o trabalho remoto seja um direito a todos os profissionais cujos cargos consigam exercê-lo. Todo trabalho que pode ser remoto deve ser remoto! Independente da localização, identidade de gênero, raça ou necessidades especiais do trabalhador.
+
+**Defender o trabalho remoto:** é importante combater os mitos, distorções, exageros e falsas notícias que o cercam.
```

Comment:
Quais mitos, distorções, etc? Seria interessante um exemplo brevíssimo ou reformular a frase para ser mais assertivo quanto ao tipo de notícia ou perspectiva individual ou coletiva acerca do trabalho remoto.